plugins/cpp_metrics/service/cxxmetrics.thrift

```diff
@@ -0,0 +1,20 @@
+include "project/common.thrift"
+include "project/project.thrift"
+
+namespace cpp cc.service.cppmetrics
+namespace java cc.service.cppmetrics
+
+enum CppMetricsType
+{
+  ParameterCount = 1,
+  McCabe = 2,
+  LackOfCohesion = 3,
+  LackOfCohesionHS = 4
+}
+
+service CppMetricsService
```

Comment:
A function returning possible metrics (id and name pairs) would also be needed.

Reply:
I added a function for this.